Detectors/TPC/qc/include/TPCQC/PID.h

```diff
@@ -18,12 +18,17 @@
 #define AliceO2_TPC_QC_PID_H
 
 #include <vector>
+#include <map>
+#include <unordered_map>
+#include <iostream>
+#include <memory>
+#include <string_view>
 
-//root includes
-#include "TH1F.h"
-#include "TH2F.h"
+// root includes
+#include "TH1.h"
+#include "TH2.h"
```

Comment:
Perhaps #inlcude "TH2.h" is required here otherwise forward declaration in helper class complains
or I include TH1F. headers in Helpers.h works
// root includes
#include "TH1F.h"
#include "TH2F.h"

//class TH1F;
//class TH2F;